packages/graph/hash_graph/bin/hash_graph/src/main.rs

```diff
@@ -64,6 +64,8 @@ async fn main() -> Result<(), GraphError> {
         tracing::info!(%account_id, "created account id");
     }
 
+    drop(connection);
+
```

Comment:
The connection now has a lifetime, so it needs to be dropped before moving the `pool` to the REST router